jib-gradle-plugin/README.md

```diff
@@ -220,12 +220,12 @@ Property | Type | Default | Description
 `volumes` | `List<String>` | *None* | Specifies a list of mount points on the container.
 `workingDirectory` | `String` | *None* | The working directory in the container.
 
-<a name="extradirectory-closure"></a>`extraDirectory` is an object with the following properties (see [Adding Arbitrary Files to the Image](#adding-arbitrary-files-to-the-image)):
+<a name="extradirectories-closure"></a>`extraDirectories` is an object with the following properties (see [Adding Arbitrary Files to the Image](#adding-arbitrary-files-to-the-image)):
 
-Property | Type
---- | ---
-`path` | `File`
-`permissions` | `Map<String, String>`
+Property | Type | Default | Description
+--- | --- | --- | ---
+`paths` | `Object` | `(project-dir)/src/main/jib` | Extra directories acceptable by [`Project.files()`](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#files-java.lang.Object...-), such as `String`, `File`, `Path`, `List<String\|File\|Path>`, etc. Can be absolute or relative to the project root.
+`permissions` | `Map<String, String>` | *None* | Maps file paths on container to Unix permissions. (Effective only for files added from extra directories.) If not configured, permissions default to 755 for directories and 644 for files.
```

Comment:
Maybe wrap the 755 and 644 in quotes, since in this context they should be strings.

Reply:
Done. Also for Gradle.